package.json

```diff
@@ -51,6 +51,7 @@
     "@babel/preset-react": "^7.9.4",
     "@types/jquery": "^3.5.10",
     "@types/lodash": "^4.14.179",
+    "@types/lodash.clonedeep": "^4.5.6",
```

Comment:
Since we don't use lodash (we only use clonedeep, merge, orderby), I suggest we remove @types/lodash and specify only @types/lodash.clonedeep. This has 0 impact on installed modules because the @types lodash/clonedeep has a dependency of lodash. But it better reflects the reality of what we use.